web_src/js/webcomponents/overflow-menu.ts

```diff
@@ -97,7 +102,16 @@ window.customElements.define('overflow-menu', class extends HTMLElement {
       const itemRight = item.offsetLeft + item.offsetWidth;
       if (menuRight - itemRight < 38) { // roughly the width of .overflow-menu-button with some extra space
         this.tippyItems.push(item);
+
+        // close tippy when clicking item of tippy
+        item.addEventListener('click', () => {
```

Comment:
You are adding duplicate event handlers, each `updateItems` call adds a bunch of new handlers.